### PR TITLE
WIP: Update README and versions for 19.09 branch 

### PR DESCRIPTION
**Pull request by dzier**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -30,192 +30,8 @@
...
-* Readiness and liveness `health endpoints
-  <https://docs.nvidia.com/deeplearning/sdk/tensorrt-inference-server-master-branch-guide/docs/http_grpc_api.html#health>`_
-  suitable for any orchestration or deployment framework, such as
-  Kubernetes.
-
-* `Metrics
-  <https://docs.nvidia.com/deeplearning/sdk/tensorrt-inference-server-master-branch-guide/docs/metrics.html>`_
-  indicating GPU utilization, server throughput, and server latency.
-
-.. overview-end-marker-do-not-remove
```
Comment (deadeyegoodwin) [inline README.rst]: You shouldn't delete the end marker or the license at the end. Notice how the README doesn't look right on github.

File: Dockerfile [hunk 2]
```diff
@@ -227,7 +227,7 @@ COPY --from=trtserver_caffe2 /opt/conda/lib/python3.6/site-packages/torch/includ
      /opt/tensorrtserver/include/torch
 COPY --from=trtserver_caffe2 /opt/conda/lib/python3.6/site-packages/torch/lib/libtorch.so \
       /opt/tensorrtserver/lib/
-COPY --from=trtserver_caffe2 /opt/conda/lib/python3.6/site-packages/torch/lib/libthnvrtc.so \
+COPY --from=trtserver_caffe2 /opt/conda/lib/python3.6/site-packages/torch/lib/libcaffe2_nvrtc.so \
```
Comment (deadeyegoodwin) [inline Dockerfile]: Note that these changes and the pytorch changes are on the r19.08 branch. They will move into the master branch when NGC releases 19.08. So when we rebase r19.09 to master we will have them automatically. I guess normally it is better to wait until after NGC release and all the steps there before doing this branch.

File: README.rst (hunk 1 above)
Comment (dzier) [reply README.rst]: I see. I can add that back in. I deleted too much based on the PR you pointed to.

File: Dockerfile (hunk 2 above)
Comment (dzier) [reply Dockerfile]: If you want, I can wait and then redo this branch.

File: README.rst (hunk 1 above)
Comment (deadeyegoodwin) [reply README.rst]: The PR doesn't delete those things. Perhaps looking at the wrong PR?

Comment (dzier) [reply README.rst]: You are correct, it doesn't. It was purely my mistake. I saw large chunks deleted and assume (incorrectly) that everything was deleted. I added back the end marker and license at the end so now it matches the PR in the example.

**Pull request closed without merging**